.travis.yml

```diff
@@ -16,14 +16,41 @@ jobs:
   - python: 3.9-dev
   - <<: *latest_py3
     env: TOXENV=docs
+    #PPC64LE
```

Comment:
This comment is unnecessary because it's obvious in what the entries below do.

Reply:
Removed.